webpack.config.js

```diff
@@ -4,7 +4,7 @@ var webpack = require('webpack')
 var HtmlWebpackPlugin = require('html-webpack-plugin')
 
 if (process.env.NODE_ENV === 'production') {
-    var publicPath = 'https://neontribe.github.io/ARCVMarket/';
```

Comment:
I don't even recall making this change (maybe it was when we were trying to get stuff to work the other day?) . Will put it back in.